materialize-mysql/client.go

```diff
@@ -189,28 +189,3 @@ func (c *client) FetchSpecAndVersion(ctx context.Context, specs sql.Table, mater
 func (c *client) Close() {
 	c.db.Close()
 }
-
-func checkIdentifierLengthError(err error) error {
-	if err == nil {
-		return nil
-	}
-
-	// Handling for errors resulting from identifiers being too long.
-	// TODO(whb): At some point we could consider moving this into `Validate` to check
```

Comment:
Finally able to clean this up, since MySQL and MariaDB both have the same limit of 64 characters for a column name.